microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -1,150 +1,281 @@
+The update policies for Microsoft Edge and the WebView2 Runtime are separate.  Even if the IT Admin has disabled automatic updates of Microsoft _Edge_, the WebView2 _Runtime_ is still automatically updated, unless the Admin turns off Runtime updating.  If the Admin disables updating Microsoft Edge (which is somewhat common), that doesn't affect which WebView2 APIs are available on the client machine.
 
-## Fixed Version distribution mode   
+<!-- de-dup: This and the previous paragraph are saying the same thing. Consider merge/deletion. -->
+If an IT Admin wants WebView2 Runtime updates to stop, the Admin must specifically set a setting.  Usually, an Admin shouldn't prevent automatically updating the WebView2 Runtime.
+<!-- /para2 -->
 
-For constrained environments with strict compatibility requirements, consider using the Fixed Version distribution mode.  Choose and package a specific version of the WebView2 Runtime using the Fixed Version distribution mode.  You may specify the timing of Runtime updates for your app.  The Fixed Version distribution mode does not receive any automatic updates. Plan to update your app and the Runtime.  
 
-> [!NOTE] 
-> The Fixed Version distribution mode was previously called bring-your-own.  
+<!-- ====================================================================== -->
+## Details about the Fixed Version Runtime distribution mode
 
-To use the Fixed Version mode, complete the following actions
+For constrained environments with strict compatibility requirements, consider using the Fixed Version distribution mode.  The Fixed Version distribution mode was previously called _bring-your-own_.
+
+To use the Fixed Version approach, get the WebView2 Runtime from [Download the WebView2 Runtime][Webview2Installer], and then package the Runtime with your WebView2 app.
+
+In the Fixed Version distribution mode, you package a specific version of the WebView2 Runtime with your WebView2 app.  You can specify the timing of updating the WebView2 Runtime for your app.<!--really?-->  In the Fixed Version distribution mode, the WebView2 Runtime on the client isn't automatically updated.  You periodically update your app together with the WebView2 Runtime.
```

Comment:
@jasonstephen15 - keep or delete this sentence?  Expand it, or link to an explanation?
"You can specify the timing of updating the WebView2 Runtime for your app."
I haven't seen this mentioned elsewhere in these 4-5 articles.  Is how to do this doc'd somewhere?